lib/scan.ts

```diff
@@ -67,6 +67,10 @@ export async function scan(options: Options): Promise<PluginResponse> {
     );
     const [filePaths, archivePaths] = await find(projectRoot, excludedPatterns);
 
+    if (!filePaths.length) {
+      throw 'No files found.';
```

Comment:
I'm not a typescript expert, but I thought the regular formatting rule for typescript exceptions was Uppercase for the first character, and no dot in the end of the message.

Reply:
![image](https://user-images.githubusercontent.com/15086861/161926528-95044134-c1cf-4f66-b1f5-9f485f5694b8.png)

I've just followed the established pattern, could change it to lower case 'no' to tighten it.